web/package.json

```diff
@@ -29,8 +29,8 @@
     "@types/react-redux": "^7.1.16",
     "@types/react-router-dom": "^5.3.0",
     "@types/webpack-env": "^1.15.1",
-    "@typescript-eslint/parser": "^5.0.0",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
```

Comment:
Why this change?

Reply:
This was most likely an accident when I was trying to use packages and deleted/reinstalled my node modules. I'm not sure if this was an actual version change or if the space it was in is just different.

Reply:
Easy to undo with `git checkout origin/main -- path/to/file`

Reply:
@JulianForeman please fix this above ^